library/catalog/templates/book-details.html

```diff
@@ -136,43 +136,45 @@ <h4 for="exampleFormControlTextarea1">Review</h4>
+                            <a style="float: right;width: 80px;"
+                               href="{% url 'change-review-status' book.id review.id 'inactive' %}"
+                               class="btn btn-danger">Delete</a>
                         {% endif %}
                     </div>
                     <div class="card-body">
                         <blockquote class="blockquote mb-0">
                             <p style="font-size: smaller">{{ review.comment }}</p>
-                            <footer class="blockquote-footer"><cite title="Source Title">Published {{ review.date }}</cite></footer>
+                            <footer class="blockquote-footer"><cite
+                                    title="Source Title">Published {{ review.date }}</cite></footer>
                         </blockquote>
                     </div>
                 </div>
             {% elif review.status == 'inactive' and user.is_authenticated and user.mongo_user.role == 'admin' or user.mongo_user.role == 'moderator' %}
                 <div style="margin-top: 5px" class="card">
                     <div style="font-size: large;font-weight: bolder" class="card-header">
                         Deleted
-                        <a style="float: right;width: 80px;" href="{% url 'change-review-status' book.id review.id 'active' %}" class="btn btn-primary">Restore</a>
+                        <a style="float: right;width: 80px;"
+                           href="{% url 'change-review-status' book.id review.id 'active' %}" class="btn btn-primary">Restore</a>
                     </div>
                     <div style="opacity: 0.5;font-size: large;font-weight: bolder" class="card-header">
                         {{ review.firstname }} {{ review.lastname }}
                     </div>
                     <div style="opacity: 0.5;" class="card-body">
                         <blockquote class="blockquote mb-0">
                             <p style="font-size: smaller">{{ review.comment }}</p>
-                            <footer class="blockquote-footer"><cite title="Source Title">Published {{ review.date }}</cite></footer>
+                            <footer class="blockquote-footer"><cite
+                                    title="Source Title">Published {{ review.date }}</cite></footer>
                         </blockquote>
                     </div>
                 </div>
             {% endif %}
 
         {% endfor %}
     </div>
-
+    <script src="{% static 'js/bookDetails.js' %}"></script>
 {% endblock %}
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
-</body>
-</html>
```

Comment:
new line

Reply:
@SerhiiPenteliuk  There is a new line, it was not before

Reply:
@SerhiiPenteliuk You can check in my file [book-details.html](https://github.com/mehalyna/Ch-117-Python-Project-ITA/blob/rating_fix/library/catalog/templates/book-details.html)